kayvee.go

```diff
@@ -27,6 +28,9 @@ const (
 
 // Format converts a map to a string of space-delimited key=val pairs
 func Format(data map[string]interface{}) string {
+	if os.Getenv("_DEPLOY_ENV") != "" {
```

Comment:
Should we have an existence check on `deploy_env` in the `data` map here? Otherwise we're going to make this syscall for every single log, right?